docs/modules/reference/pages/configuration/tuning-activemq.adoc

```diff
@@ -50,7 +50,35 @@ In order to isolate multiple instances on the same broker, you can customize the
 echo 'org.opennms.instance.id=MyNMS' > "$\{OPENNMS_HOME}/etc/opennms.properties.d/instance-id.properties"
 ----
 
-CAUTION: If you change the instance id setting when using the embedded broker, you will need to update the authorization section in the broker's configuration to reflect the updated prefix.
+Update minion's instance id accordingly to match the OpenNMS instance's. The same id instance property should be present in ${MINION_HOME}/etc/custom.system.properties.
+
+[source, sh]
+----
+echo 'org.opennms.instance.id=MyNMS' > "$\{MINION_HOME}/etc/custom.system.properties"
+----
+CAUTION: If you change the instance id setting when using the embedded broker, you will need to update the authorization section in the broker's configuration to reflect the updated prefix:
```

Comment:
Maybe update the note to remind the user of the path to file where this change should go.

Reply:
I put ${MINION_HOME} here on purpose to make it more generic as the path depends on the distribution so that it can appear not clunky. Let me know if you think it's still better to include the reminder!

Reply:
I was referring to where the XML code goes for the ActiveMQ settings.

Reply:
Thanks @mmahacek for pointing that out! I've updated it. Hopefully it's clear now.